src/Features/VisualBasic/Portable/CodeRefactorings/AddMissingImports/VisualBasicAddMissingImportsRefactoringProvider.vb

```diff
@@ -0,0 +1,24 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.AddMissingImports
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.PasteTracking
+Imports Microsoft.CodeAnalysis.VisualBasic
+
+<ExportCodeRefactoringProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeRefactoringProviderNames.AddMissingImports), [Shared]>
+Friend Class VisualBasicAddMissingImportsRefactoringProvider
+    Inherits AbstractAddMissingImportsRefactoringProvider
+
+    Protected Overrides ReadOnly Property CodeActionTitle As String
+        Get
+            Return VBFeaturesResources.Add_missing_Imports
+        End Get
+    End Property
```

Comment:
fyi, vb props can be written as:     Protected Overrides ReadOnly Property CodeActionTitle As String = ...

It's much simpler :)